app/src/main/res/layout/placeholder_item_card_tickets.xml

```diff
@@ -10,57 +10,74 @@
         android:layout_width="match_parent"
         android:layout_height="@dimen/avatar_medium"
         android:orientation="horizontal">
 
         <View
+            android:id="@+id/viewLeft"
             android:layout_width="@dimen/card_width_45dp"
             android:layout_height="@dimen/card_width_45dp"
+            android:layout_marginStart="@dimen/layout_margin_medium"
             android:layout_marginLeft="@dimen/layout_margin_medium"
             android:layout_marginTop="@dimen/layout_margin_medium"
-            android:background="@color/shimmer_background" />
+            android:background="@color/shimmer_background"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
 
-        <LinearLayout
-            android:layout_width="@dimen/item_image_view_160dp"
-            android:layout_height="match_parent"
-            android:layout_marginLeft="@dimen/layout_margin_small"
-            android:layout_marginRight="@dimen/layout_margin_small"
-            android:orientation="vertical"
-            android:padding="@dimen/padding_medium">
-
-            <View
-                android:layout_width="match_parent"
-                android:layout_height="@dimen/layout_margin_none"
-                android:layout_marginBottom="@dimen/layout_margin_small"
-                android:layout_weight="1"
-                android:background="@color/shimmer_background"
-                android:ellipsize="end" />
-
-            <View
-                android:layout_width="match_parent"
-                android:layout_height="@dimen/layout_margin_none"
-                android:layout_marginBottom="@dimen/layout_margin_small"
-                android:layout_weight="1"
-                android:background="@color/shimmer_background"
-                android:ellipsize="end" />
+        <View
+            android:id="@+id/view1"
+            android:layout_width="0dp"
+            android:layout_height="30dp"
+            android:layout_marginLeft="@dimen/layout_margin_medium"
+            android:layout_marginRight="@dimen/layout_margin_medium"
+            android:layout_marginBottom="@dimen/layout_margin_small"
+            android:layout_marginTop="@dimen/details_margin_small"
+            android:background="@color/shimmer_background"
+            android:ellipsize="end"
+            app:layout_constraintLeft_toRightOf="@+id/viewLeft"
+            app:layout_constraintRight_toLeftOf="@+id/viewRight"
+            app:layout_constraintTop_toTopOf="parent" />
 
-            <View
-                android:layout_width="match_parent"
-                android:layout_height="@dimen/layout_margin_none"
-                android:layout_weight="1"
-                android:background="@color/shimmer_background"
-                android:ellipsize="end" />
+        <View
+            android:id="@+id/view2"
+            android:layout_width="0dp"
+            android:layout_height="30dp"
```

Comment:
Define all dimensions in dimenstion resources file.

Reply:
@liveHarshit Done